index.html

```diff
@@ -0,0 +1,469 @@
+      <div class="proj"></div>
+      <div id="victor">
+        <img src="image/victor.png" alt="" id="vic" />
+      </div>
+    </section>
+    <div class="Desktop-project-section">
+      <h2 id="project-main-header">Projects</h2>
+      <div id="slider-container"></div>
+    </div>
+    <!-- contact part -->
+    <section class="contact" id="contact">
+      <div class="contact-container">
+        <h2>Contact <span>Me</span></h2>
+        <div class="line"></div>
+        <form id="contact-form">
+          <div class="form-group">
+            <div class="input-wrapper">
+              <label for="name">Name</label>
+              <input
+                type="text"
+                id="name"
+                placeholder="Type your name"
+                required
+              />
+              <div
+                id="name-feedback"
+                class="feedback hidden"
+                aria-live="assertive"
+              ></div>
+            </div>
+            <div class="input-wrapper">
+              <label for="last-name">Last Name</label>
+              <input
+                type="text"
+                id="last-name"
+                placeholder="Type your last name"
+                required
+              />
+              <div
+                id="last-name-feedback"
+                class="feedback hidden"
+                aria-live="assertive"
+              ></div>
+            </div>
+          </div>
+
+          <div class="form-group">
+            <div class="input-wrapper">
+              <label for="email">Email</label>
+              <input
+                type="email"
+                id="email"
+                placeholder="Type your email"
+                required
+              />
+              <div
+                id="email-feedback"
+                class="feedback hidden"
+                aria-live="assertive"
+              ></div>
+            </div>
+            <div class="input-wrapper">
+              <label for="phone-number">Phone Number</label>
+              <input
+                type="tel"
+                id="phone-number"
+                placeholder="Type your phone number"
+                required
+              />
+              <div
+                id="phone-number-feedback"
+                class="feedback hidden"
+                aria-live="assertive"
+              ></div>
+            </div>
+          </div>
+
+          <div class="input-wrapper">
+            <label for="message">Your Message</label>
+            <textarea
+              id="message"
+              cols="30"
+              rows="10"
+              placeholder="Type your message"
+              required
+            ></textarea>
+            <div
+              id="message-feedback"
+              class="feedback hidden"
+              aria-live="assertive"
+            ></div>
+          </div>
+
+          <button id="submit-btn" type="submit">Submit</button>
```

Comment:
- The error messages are a little bit below their field, please make it near their exact field:
- For the phone number you should not check for 10 digits as maybe the user is from a country with more than 10 digits number, for example, Germany.
![image](https://github.com/user-attachments/assets/323bb4b7-08f8-4e4a-a618-e2cc471d8881)